modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -307,10 +307,26 @@ public void processOpts() {
                 }
             }
         }
+
+        processUsings(objs);
+
         // process enum in models
         return postProcessModelsEnum(objs);
     }
 
+    private void processUsings(Map<String, Object> objs) {
```

Comment:
Hi @jimschubert I did wonder about that solution, fully qualifying all objects, however as far as I can see all if the imports, at least running when my tests against https://api.bitbucket.org/swagger.json have to be in the form "Model.ClassName" unless there is a configuration overriding the import mapping. It doesn't appear possible to have an import in the form System.IO.

To be fair I am new to Swagger so I may be missing something